src/reputation/views/withdrawal_view.py

```diff
@@ -140,9 +85,9 @@ def create(self, request):
                 status=400,
             )
 
-        if user.reputation < 110:
+        if user.author_profile.get_rep_score() < 10:
```

Comment:
@koutst we also need a condition for users that have their identity verified. So this should be:

`if not (sufficient_rep or user_is_verified): Error`